packages/common/string.js

```diff
@@ -25,12 +45,94 @@ const unicodeEscape = (str, regex = /[\s\S]/g) => str.replace(regex, unicodeEsca
  * @param {string} oldSubString - The string to replace
  * @param {string} newSubString - The string replacement
  * @returns {string} the modified string
+ *
+ * @static
  */
 function globalReplace(string, oldSubString, newSubString) {
   return string.replace(new RegExp(oldSubString, 'g'), newSubString);
 }
 
+/**
+ * Converts string, as a whole, to lower case just like String#toLowerCase
+ *
+ * @param {string} str - the string to convert
+ * @returns {string} the lower-cased string
+ *
+ * @static
+ */
+const toLower = (str) => str.toLowerCase();
```

Comment:
Do all these new functions have unit tests?

Reply:
Added in 3cdaef3d1